en/docs/learn/configuring-oidc-federated-idp-initiated-logout.md

```diff
@@ -0,0 +1,167 @@
+
+1. Run the WSO2 Identity Server on port 9443 (Primary IS).
+2. Log in to the management console as an administrator using the admin,admin credentials.
+3. Navigate to **Main** to access the **Identity** menu.
+4. Click **Add** under **Identity Providers**.
+5. Fill in the details in the **Basic Information** section. Give a suitable name for the Identity Provider.
+6. Expand the **OAuth2/OpenID Connect Configuration** section under **Federated Authenticators** section.
+7. Fill the fields as follows.
+
+Here client id and secret is the Oauth Client Key and Secret generated in the above step.
+
+- Authorization Endpoint URL - https://localhost.com:9444/oauth2/authorize
+- Token Endpoint URL - https://localhost.com:9444/oauth2/token
+- Callback Url - https://localhost.com>:9443/commonauth
+- Userinfo Endpoint URL - https://localhost.com:9444/oauth2/userinfo
+- Logout Endpoint URL - https://localhost.com:9444/oidc/logout
+
+Add `scope=openid` in the **Additional Query Parameters**.
+
+![oidc-fed-idp-config-in-primary-idp](../assets/img/tutorials/oidc-fed-idp-config-in-primary-idp.png)
+
+8. Signature of the logout token is validated using either, using registered JWKS uri or uploaded certificate to the
+   relevant identity provider.
+
+   - Under the **Basic Information** section select the **Use IDP JWKS endpoint** option from **Choose IDP certificate
+     type** and add the JWKS uri
+     `https://localhost.com:9444/oauth2/jwks` to Identity Provider's JWKS Endpoint.
+     ![oidc-primary-idp-certificate-config](../assets/img/tutorials/oidc-primary-idp-certificate-config.png)
+
+    - Alternatively, select the **Upload IDP certificate** option from **Choose IDP certificate type** and upload the
+      certificate of the SecondaryIS.
+      ![oidc-primary-idp-jwks-uri-config](../assets/img/tutorials/oidc-primary-idp-jwks-uri-config.png)
+
+9. Click **Register**.
+
+### Configuring Pickup Dispatch application in the Primary IS
+
+1. Follow the steps
+   in [Deploying the pickup-dispatch webapp](https://is.docs.wso2.com/en/latest/learn/deploying-the-sample-app/#deploying-the-pickup-dispatch-webapp)
+   to download, deploy and register the **Pickup-Dispatch** sample.
+2. Once you have added the OIDC service provider, go to the **Service Provider Configuration** and expand **Local &
+   Outbound Authentication Configuration**.
+3. Select **Federated Authentication** and from the dropdown menu select the **SecondaryIS**. Click **Update**.
+
+   ![oidc-service-provider-federated-authentication](../assets/img/tutorials/oidc-service-provider-federated-authentication.png)
+
+### Configuring Pickup Manager application in the Secondary IS
+
+Follow the steps
+in [Deploying the pickup-manager webapp](https://is.docs.wso2.com/en/latest/learn/deploying-the-sample-app/#deploying-the-pickup-manager-webapp)
+to download, deploy and register the Pickup-Manager sample.
+
+## OIDC Back-channel Logout Token Validation
+
```

Comment:
can we have a sample of the logout token here and also links it to back-channel logout doc?